### DISCUSSION TITLE
how to use voice recording?

### DISCUSSION THREAD

**Discussion by ghost** [Pycord-Development/pycord#1047] [Q&A] [answered]
i've been testing the function `voice_client.start_recording`, and i managed to get the files
but, it returns multiple audio files, one for each user, with different length, making it hard to syncronize

and i didn't found anything in the documentation explaining how to use it properly
i need the audio syncronized, like making all the files having the same length, or save in a single file (like a multi-track mp4/mkv)

**Comment by VincentRPS** [comment, marked as answer]
Sadly this is a hard thing to do since discord sends audio data per-user meaning we can’t just give all the audio data, on the other hand we have tried to make filters and such for synchronizing the audio but never really got far. The best hope might be to manually go into an editor and stuff the audio together.

**Reply by ghost** [reply]
it get's worse to sync if an user disconnects and reconnects
syncronizing in an editor would be painful
